### ISSUE TITLE
vscode error

### ISSUE THREAD

**Issue by elvizlai** [mgechev/revive#31]
**Describe the bug**
```
Error while running tool: /Users/elvizlai/.godeps/bin/revive -config=/Users/elvizlai/.revive.toml -exclude=vendor/... -formatter friendly
cannot read the config file
```

**To Reproduce**
1. I updated revive `go get -u github.com/mgechev/revive`
2. I run it with the following flags & configuration file:

vscode user config
```
    "go.lintTool": "revive",
    "go.lintFlags": [
        "-config=~/.revive.toml -exclude=vendor/... -formatter friendly"
    ]
```

```shell
# flags

revive ...
```

```toml
ignoreGeneratedHeader = false
severity = "warning"
confidence = 0.8
errorCode = 0
warningCode = 0

[rule.blank-imports]
[rule.context-as-argument]
[rule.context-keys-type]
[rule.dot-imports]
[rule.error-return]
[rule.error-strings]
[rule.error-naming]
# [rule.exported]
[rule.if-return]
[rule.increment-decrement]
# [rule.var-naming]
[rule.var-declaration]
[rule.package-comments]
[rule.range]
[rule.receiver-naming]
[rule.time-naming]
[rule.unexported-return]
[rule.indent-error-flow]
[rule.errorf]
```

**Expected behavior**
I run the cmd manual, it works. but not the vscode

**Logs**
```
Error while running tool: /Users/elvizlai/.godeps/bin/revive -config=/Users/elvizlai/.revive.toml -exclude=vendor/... -formatter friendly
cannot read the config file
```

**Desktop (please complete the following information):**
 - OS: mac os x
 - Version of Go
latest

**Additional context**
Nothing

**Comment by mgechev** [comment]
@elvizlai would you share what the cause is?

**Comment by elvizlai** [comment]
@mgechev my mistake. the `go.lintFlags` should be an array. each flag should be separated.

**Comment by mgechev** [comment]
Thanks for sharing this! I hope it helps others who face the same problem.
